tests/pipelines/test_api.py

```diff
@@ -171,13 +174,13 @@ def test_create(pipelines_api):
     spec = copy.deepcopy(SPEC_WITHOUT_ID)
     spec['libraries'] = []
 
-    pipelines_api.create(spec, allow_duplicate_names=False)
+    pipelines_api.create(spec, spec_dir='.', allow_duplicate_names=False)
```

Comment:
If all these tests assume that the spec dir is the same as the current dir, would any of them actually fail even without the code changes?

Reply:
these tests use a spec without libraries, so it makes no difference to them.

Reply:
So which one's the test that would fail without the prod code changes?

Reply:
This one: https://github.com/databricks/databricks-cli/pull/368/files#diff-66dd45a6659dae8bccf642b5526207c7af3ab20f872109aa759bd9fbfd76b070R88